extra_docs/relations/README.md

```diff
@@ -0,0 +1,163 @@
+# Relations
+All project resources can be related to each other and SDK supports inserting, fetching and attaching/detaching related 
+records.
+Let's create a simple schema which will be used for all further examples:
+
+![Relations schema](https://jexia.github.io/jexia-sdk-js/assets/relations-schema.png) 
+
+There are three datasets and two relations: **posts** has *one-2-many* relation to **comments** and **comments** has 
+*one-2-one* relation to **author** (remember that all management opearations, such as creating datasets, fields and 
```

Comment:
Is there any chance to show example with UMS?

Reply:
Not yet, we didn't implement relations to UMS so far

Reply:
as I know it is supported from BE. Or you mean not supported in jsSDK?

Reply:
UMS does not support any CRUD operations, there is no insert, update or select, no conditions, no filters. I do not see any way how to add relations there